src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -222,7 +222,11 @@ const Conversation: React.FC<ConversationProps> = props => {
           </Flex>
         </Box>
       </MessageContainer>
-      <Reply conversation={conversation} environment={relay.environment} />
+      <Reply
+        handleScroll={scrollToBottom}
```

Comment:
When passing event handlers in as props the prop is typically named `onSomething`. A good example is how click handlers, etc, on divs are always `onClick`, `onPress`, etc. 

The `handle` naming typically applies to functions inside the component, eg, 

```ts
const handleClick = () => ...

<div onClick={handleClick}>
```